content/docs/start/experiments-trail/experiments.md

```diff
@@ -0,0 +1,319 @@
+---
+title: 'Get Started: Experiments'
+---
+
+# Get Started with Experiments
+
+<abbr>Experiments</abbr> proliferate quickly in ML projects where there are many
+parameters to tune or other permutations of the code. We can organize such
+projects and keep only what we ultimately need with `dvc experiments`. DVC can
+track experiments for you so there's no need to commit each one to Git. This way
+your repo doesn't become polluted with all of them. You can discard experiments
+once they're no longer needed.
+
+Previously, we learned how to tune [ML pipelines](/doc/start/data-pipelines) and
+[compare the changes](/doc/start/metrics-parameters-plots). In this section, we
+will explore the basic features of DVC experiment management with
```

Comment:
can we explain this from the "what users get with this " perspective? E.g. fast local iterations and ability to pick the best? or something like this? otherwise we explain Experiments with experiments management - it's not very informative I'm afraid.